lib/diagnostic.js

```diff
@@ -7,23 +7,43 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-const min = function min() {
-
+const min = function min(nums)
+{
+  let result;
+  for(let i in nums)
+  {
+    if (result > nums[i]) {result = nums[i];}
+  }
+  return result;
 };
```

Comment:
Also, your function is not capturing multiple arguments passed in. Do you remember `arguments`? Check out this solution:
```
const min = function min() {
  let result = arguments[0];
  for (let i = 1; i < arguments.length; i++) {
    if (arguments[i] < result) {
      result = arguments[i];
    }
  }
  return result;
};
```